tests/unittests/OperatorTest.cpp

```diff
@@ -1241,31 +1256,43 @@ TEST_P(OperatorTest, Gather32) {
   EE_.compile(CompilationMode::Infer, F_);
   EE_.run(bindings_);
 
-  auto H = bindings_.get(result->getPlaceholder())->getHandle();
+  auto H = bindings_.get(result->getPlaceholder())->getHandle<int8_t>();
 
-  EXPECT_FLOAT_EQ(H.at({0, 0, 0}), 1.0);
-  EXPECT_FLOAT_EQ(H.at({0, 0, 1}), 1.2);
-  EXPECT_FLOAT_EQ(H.at({0, 1, 0}), 2.3);
-  EXPECT_FLOAT_EQ(H.at({0, 1, 1}), 3.4);
-  EXPECT_FLOAT_EQ(H.at({0, 2, 0}), 1.0);
-  EXPECT_FLOAT_EQ(H.at({0, 2, 1}), 1.2);
-  EXPECT_FLOAT_EQ(H.at({0, 3, 0}), 2.3);
-  EXPECT_FLOAT_EQ(H.at({0, 3, 1}), 3.4);
+  EXPECT_EQ(H.at({0, 0, 0}), 1);
```

Comment:
Consider creating an expected result `Tensor`, initializing it with the values in `OUTPUT` in the comment above, and then replacing this series of `EXPECT_EQ` statements with either a for-loop or a call to `Tensor::isEquals`: